### DISCUSSION TITLE
Usuario do tipo Participante precisa de controle de acesso (usuario/senha)?

### DISCUSSION THREAD

**Discussion by jenrodrigues** [andrebaltieri/AgendaEventos#10] [Q&A] [answered]
Nao sei se isso ja foi discutido, mas o Participante vai precisar informar usuario e senha quando se inscrever em um evento ou quando remover sua inscricao?

**Comment by renebentes** [comment]
Creio que sim. Como vou ter acesso aos eventos que participei?

**Comment by BrewertonSantos** [comment]
Se bem me lembro já tocamos no assunto @jenrodrigues . Por enquanto um link é gerado para uma área externa. Isso foi uma das limitações atuais para não conseguirmos gerarmos mais informações úteis para pensar em outras funcionalidades. Mas que está aberto para próximas etapas.

**Comment by LeonardoPedrosa** [comment]
Acredito que deveríamos fechar o link de acesso ao evento pela plataforma para usuários não conectados.
A exemplo do Meetup, o link apenas fica visível para quem estiver logado na plataforma, as demais informações ficam acessíveis justamente como porta de entrada para esses cadastros.
![image](https://user-images.githubusercontent.com/21269291/116233132-523e9000-a731-11eb-8dc6-5ce02fc93bef.png)

**Comment by nilson-teodoro** [comment]
No meu entendimento sim, se não, fica difícil monitorar quais eventos participante X está inscrito.

**Comment by jenrodrigues** [comment, marked as answer]
Decidimos que SIM, e necessario criar controle de acesso para Participantes.
